airportDeparture_CraigBelinfante.playground/Contents.swift

```diff
@@ -0,0 +1,126 @@
+import UIKit
+
+
+enum FlightStatus: String {
+    case EnRoute = "En Route"
+    case Landed = "Landed"
+    case Scheduled = "Scheduled"
+    case Delayed = "Delayed"
+    case Canceled = "Canceled"
+    case Boarded = "Boarding"
+}
+
+struct Airport {
+    let name: String
+    let symbol: String
+}
+
+struct Flight {
+    let destination: String
```

Comment:
destination should be of type Airport
```suggestion
    let destination: Airport
```